pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -81,6 +81,11 @@ func (c *Cluster) checkHealth() error {
 	c.acquireOrchestrationLock()
 	defer c.releaseOrchestrationLock()
 
+	if c.spec.Mon.Count == 0 || c.ClusterInfo == nil {
+		logger.Warningf("skipping mon health check since cluster info is not initialized")
```

Comment:
The warning is not 100% accurate since the condition is `OR`, we could enter it if `c.spec.Mon.Count == 0` or perhaps you mean to use `AND`?

Reply:
"cluster info" was meant to be a generic message about not being initialized. The log message doesn't need all the details of the check.